### DISCUSSION TITLE
Is it possible to skip the indentation for the first level?

### DISCUSSION THREAD

**Discussion by nadropper** [lukas-reineke/indent-blankline.nvim#565] [Q&A] [answered]
Hello, there.

Is it possible to skip the first indentation level to get something like the screenshot below?

<img width="1680" alt="Screenshot 2023-04-23 at 5 31 18 PM" src="https://user-images.githubusercontent.com/49292621/233869622-b8d4765b-c8a9-45a3-9e7f-6c65060ab35a.png">

**Comment by lukas-reineke** [comment, marked as answer]
Yes, `:help g:indent_blankline_show_first_indent_level`

**Reply by nadropper** [reply]
Thanks, @lukas-reineke!
